### PR TITLE
post: 2024.12 announcement

### PR DESCRIPTION
**Pull request by mika**
Thanks Chris for providing the initial draft in
https://github.com/grml/blog.grml.org/pull/5

### CODE REVIEW

**Review by zeha — APPROVED** [review]

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+With release version 2024.12 we announce our first release supporting 64-bit ARM CPUs (Architecture `arm64`)!
+We would like to thank [netcup](https://www.netcup.com/) for their financial support in making this possible.
+
+At the same time we want to point out that releases for [32-bit x86 PCs have been discontinued]({{< ref "416-grml32-sunset" >}}).
+With the deprecation of the grml32 flavor, we instead implemented UEFI 32-bit boot support on our Grml amd64 flavor.
+
+For detailed information about the changes between 2024.02 and 2024.12 have a look at [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).
+
+Thanks to everyone contributing to Grml and this release, stay healthy and happy Grml-ing!
```
Comment (jkirk) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]: This feels not soo good too me. What about:

"Thanks to everyone who contributed to Grml and this release, [...]" or
"Thanks to everyone for their contributions to Grml and this release, [...]"
(with the help of DeepL)

Everything else LGTM! Thx!

Comment (mika) [reply content/post/418-Grml-new-stable-release-2024.12-available.md]: ack, thx - done

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+slug: 418-Grml-new-stable-release-2024.12-available
+title: Grml - new stable release 2024.12 available
+
+---
+We are proud to announce the new stable release version 2024.12, code-named ['Adventgrenze'](https://grml.org/faq/#releasename)!
+
+This Grml release provides fresh software packages from Debian trixie.
+As usual it also incorporates current hardware support and fixes known bugs from the previous Grml release.
+
+With release version 2024.12 we announce our first release supporting 64-bit ARM CPUs (Architecture `arm64`)!
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
With version 2024.12 Grml, for the first time ever, supports 64-bit ARM CPUs (Architecture `arm64`)!
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+author: Michael Prokop
+categories: []
+date: Fri, 20 Dec 2024 11:42:23 +0100
+layout: post
+slug: 418-Grml-new-stable-release-2024.12-available
+title: Grml - new stable release 2024.12 available
+
+---
+We are proud to announce the new stable release version 2024.12, code-named ['Adventgrenze'](https://grml.org/faq/#releasename)!
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
We are proud to announce our new stable release version 2024.12, code-named ['Adventgrenze'](https://grml.org/faq/#releasename)!
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+This Grml release provides fresh software packages from Debian trixie.
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
This Grml release brings you fresh software packages from Debian trixie, enhanced hardware support and addresses known bugs from previous releases.
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+As usual it also incorporates current hardware support and fixes known bugs from the previous Grml release.
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+For the first time ever, version 2024.12 supports 64-bit ARM CPUs (Architecture `arm64`)!
+We would like to thank [netcup](https://www.netcup.com/) for their financial support in making this possible.
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
This milestone was made possible thanks to the financial support from [netcup](https://www.netcup.com/).
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+At the same time we want to point out that releases for [32-bit x86 PCs have been discontinued]({{< ref "416-grml32-sunset" >}}).
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
As previously announced, releases for [32-bit x86 PCs have been discontinued]({{< ref "416-grml32-sunset" >}}).
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+With the deprecation of the grml32 flavor, we instead implemented UEFI 32-bit boot support on our Grml amd64 flavor.
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
With the grml32 flavor removed, we've introduced UEFI 32-bit boot support on our Grml amd64 flavor to ensure 64-bit PCs with 32-bit Firmware can use this release.
```

File: content/post/418-Grml-new-stable-release-2024.12-available.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+For detailed information about the changes between 2024.02 and 2024.12 have a look at [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).
```
Comment (zeha) [inline content/post/418-Grml-new-stable-release-2024.12-available.md]:
```suggestion
For a detailed overview of the changes from 2024.02 to 2024.12, please check out [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).
```

Comment (mika) [reply content/post/418-Grml-new-stable-release-2024.12-available.md]: ACK, geat suggestions, applied all of them!

**Review by zeha — APPROVED** [review]

**Pull request merged**